### PR TITLE
Add minutes for 2018-08-22

### PR DESCRIPTION
**Pull request by BKiddo**

### CODE REVIEW

File: 2018/2018-08-22.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### Specific:
+-Kajetan will get quotes from the potential venues and check our needs can be met.
+-Dan and Jake will review the website.
+-Andy BS will take ownership of volunteer announcements.
+-Andy B will send a volunteer template to STEM abassadors.
+-Andy BS will email Jimmy regarding team list access, on behalf of Jake.
+-Andy BS will add a request to access budget in his email to Jimmy.
+-Kajetan will draw up inventory system pros and cons.
+
```
Comment (RealOrangeOne) [inline 2018/2018-08-22.md]: Lots of trailing new lines here

File: 2018/2018-08-22.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	   Kajetan will draw up system pros and cons.
+	2. The weekend of 22nd Sept is suggested for a kit session but this is dependent on being able to access the container.
+
+## Action Points:
+
+### General:
+-The core team will make a dedicated venues slack channel.
+
+### Specific:
+-Kajetan will get quotes from the potential venues and check our needs can be met.
```
Comment (RealOrangeOne) [inline 2018/2018-08-22.md]: Need a space after the `-`, otherwise they won't be picked up as bullet points

File: 2018/2018-08-22.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Core Team Meeting 2018-08-22 19:00
+
+## Attendees:
+	Present:
```
Comment (RealOrangeOne) [inline 2018/2018-08-22.md]: Try using sub-headings for present/apologies here, and unindent the bullet list.

File: 2018/2018-08-22.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- Andy Busse
+- Alistair Lynn
+- Tyler Ward
+- Dan Trickey
+- Kajetan Champlewski
+- Andy Barrett-Sprock
+- Jake Howard
+- Antoine Petty
+- Holly Holder (minutes)
+ # Apologies:
```
Comment (RealOrangeOne) [inline 2018/2018-08-22.md]: Looks like there's a leading space on this line. Probably also wants a blank line above it to separate it from the previous section

Comment (BKiddo) [reply 2018/2018-08-22.md]: Removed space but disagree with separation as all part of "attendees" section

File: 2018/2018-08-22.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+# Core Team Meeting 2018-08-22 19:00
+
+## Attendees:
+# Present:
+- Andy Busse
+- Alistair Lynn
+- Tyler Ward
+- Dan Trickey
+- Kajetan Champlewski
+- Andy Barrett-Sprock
```
Comment (trickeydan) [inline 2018/2018-08-22.md]: I believe the accepted spelling of @Adimote's name is ~~Andles Bandles-Sport~~ Andy Barrett Sprot

File: 2018/2018-08-22.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+	6. Schools recruitment
+	7. Kickstart plan
+	8. Budget
+3. AOB 
+	1. Southampton cupboard
+
+## Minutes: 
+1. To focus and condense future meeting the core team agree to make use of GitHub Issues and slack for minor decisions.
+2. 
+	1. Alistair has drawn up a detailed list of venue requirements. Cureently there is a shortlist of a current possible venues, including Newbury and Farnborough.
```
Comment (trickeydan) [inline 2018/2018-08-22.md]: Typo here: s/Cureently/Currently/

File: 2018/2018-08-22.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+	   Kajetan will get quotes from the potential venues and check our needs can be met.
+	   The core team will make a dedicated venues slack channel.
+	2. Andy still has not heard back re. the storage container.
+	3. Peter has not yet made decisions regarding Saffron etc. 
+	   The core team agree to postpone a final decision regarding the IDE.
+	   The core team note that the websit should be updates and the documentation reviewed.
```
Comment (trickeydan) [inline 2018/2018-08-22.md]: Another typo: s/websit/website/

File: 2018/2018-08-22.md (hunk 5 above)
Comment (Adimote) [reply 2018/2018-08-22.md]: Andy Brussels-Sprout is also a valid spelling ;)

**Pull request merged**